### ISSUE TITLE
KeyError when laptop screen is closed

### ISSUE THREAD

**Issue by IFD3** [Crozzers/screen_brightness_control#33]
**Describe the bug**
I get a KeyError at any function when having a laptop closed so the screen is off.

**To Reproduce**
Having a Laptop with a screen closed and 2 other screens connected that are in use.
After initializing the class and trying to use any function that uses the get_display_info

**Expected behavior**
Screen should be excluded

**Debug info**
DEBUG:screen_brightness_control._debug.info:gathering list of all monitors
DEBUG:screen_brightness_control.helpers.__Cache_2307454344528:cache get 'windows_monitors_info_raw' = [KeyError]
DEBUG:screen_brightness_control._debug.info:filtering the list of monitors
DEBUG:screen_brightness_control._debug.info:getting display info for method: wmi
DEBUG:screen_brightness_control.helpers.__Cache_2307454344528:cache get 'windows_monitors_info_raw' = [KeyError]
DEBUG:screen_brightness_control._debug.info:getting display info for method: vcp
DEBUG:screen_brightness_control.helpers.__Cache_2307454344528:cache get 'windows_monitors_info_raw' = [KeyError]
DEBUG:screen_brightness_control._debug.info:windows specific debug info
DEBUG:screen_brightness_control:brightness get request display None with method None
DEBUG:screen_brightness_control.helpers.__Cache_2307454344528:cache get 'windows_monitors_info_raw' = [KeyError]

**Additional context**
  File "screenBrightness.py", line 31, in __init__
    for monitor in sbc.list_monitors_info():
                   ^^^^^^^^^^^^^^^^^^^^^^^^
  File ".venv\Lib\site-packages\screen_brightness_control\__init__.py", line 256, in list_monitors_info
    return _OS_MODULE.list_monitors_info(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File ".venv\Lib\site-packages\screen_brightness_control\windows.py", line 369, in list_monitors_info
    info = get_display_info()
           ^^^^^^^^^^^^^^^^^^
  File ".venv\Lib\site-packages\screen_brightness_control\windows.py", line 92, in get_display_info
    pydevice = monitor_uids[instance_name]
               ~~~~~~~~~~~~^^^^^^^^^^^^^^^
**KeyError: '4&1319fe32&1&UID8388688'**

Checking the dict it is obvious the key is in the **WMI** list but somehow not in the **monitor_uids** or rather the return of **enum_display_devices()** 

{'4&1319fe32&1&UID78739': <PyDISPLAY_DEVICE object at 0x0000017C09085970>, '4&1319fe32&1&UID4361': <PyDISPLAY_DEVICE object at 0x0000017C09085CE0>}

4&1319fe32&1&UID2288688

I can think of different solutions to fix this, but since I see different approaches in the code already using loops over displays I rather report it than messing around in the code myself.

**Comment by Crozzers** [comment]
Thanks for this!

Interestingly, the error doesn't appear if you start Windows with the laptop closed. It's only if the laptop screen is opened at any point and closed again that the error crops up.

I've added a fix over on the [dev branch](https://github.com/Crozzers/screen_brightness_control/tree/dev) so if you could check the fix works that would be great!

**Comment by Crozzers** [comment]
Released [v0.22.1](https://github.com/Crozzers/screen_brightness_control/releases/tag/v0.22.1) with the fix

**Comment by IFD3** [comment]
oh yeah, I couldnt check in time but I will soon when I return to the project I am using it.
